### PR TITLE
Fix ContinuesEquation

### PR DESCRIPTION
**Pull request by MrBrain295**
## Adding a language

- [ ] The code displays "Hello World"
- [ ] I have updated the readme to include the new language
- [ ] I have incremented the language count in the readme
The above don’t apply

### CODE REVIEW

**Comment by Richienb** [comment]
Looks like there is also a comma

**Comment by MrBrain295** [comment]
Done @Richienb

File: c/ContinuesEquation.ce [hunk 1]
```diff
@@ -1 +1 @@
-0 0 o'H'e'l'l'o',' 'w'o'r'l'd'!
+0 0 o'H'e'l'l'o' 'W'o'r'l'd'
```
Comment (Richienb) [inline c/ContinuesEquation.ce:1]: Each single quote should be followed by an ascii character.

```suggestion
0 0 o'H'e'l'l'o' 'W'o'r'l'd
```

**Review by Richienb — CHANGES_REQUESTED** [review]
The file appears to be incorrectly named and needs to be renamed from `ContinuesEquation` to [`ContinuousEquation`](https://esolangs.org/wiki/ContinuousEquation).

**Pull request merged**